patterns/part_of.yaml

```diff
@@ -17,6 +17,11 @@ annotations:
   - annotationProperty: part_of_some
     var: anatomical_entity
 
+name:
+  text: "part of some %s"
```

Comment:
So we want `some` in the label?

Reply:
For these I just went with what felt right. Except for 'absence', these will all be disappearing when we switch to relation graphs.